src/components/shared/MessageRenderer.tsx

```
@@ -132,17 +132,17 @@ const MarkdownComponents = {
           });
           return (
             <span 
-              className="inline-flex items-center bg-blue-50 text-blue-800 px-2 py-1 rounded text-sm border border-blue-200 mx-0.5"
+              className="inline-flex items-center bg-blue-50 dark:bg-blue-900/30 text-blue-800 dark:text-blue-300 px-2 py-1 rounded text-sm border border-blue-200 dark:border-blue-700 mx-0.5"
               dangerouslySetInnerHTML={{ __html: mathMLString }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)